wgsl/index.bs

```diff
@@ -306,6 +306,11 @@ This is a portability hazard, as different implementations may exhibit the diffe
 The design of WGSL aims to minimize such cases, but is constrained by feasibility,
 and goals for achieving high performance across a broad range of devices.
 
+WGSL sometimes imposes a <dfn noexport>behavioral requirement</dfn> on
+implementations.
+These are behaviors an implementation must satisfy in order to be conformant
+with the specification.
```

Comment:
Is this the kind of thing you wanted?

>   Behavioral requirements are actions the implementation will perform when processing or executing a WGSL program.  They describe the implementation's obligations in the contract with the programmer.

Reply:
I was thinking more in lines of motivation. In a way the whole spec is a set of behavioral requirements, but we call out certain ones because they are special in some way. Improving portability or guaranteeing something not provided by underlying platforms.

Reply:
I've tried improving the language by building on your suggestion.